src/types/index.d.ts

```diff
@@ -16,3 +17,15 @@ export interface Config {
   wallet: Wallet;
   forms: Form[];
 }
+
+interface Attachments {
+  allow: boolean;
+  accept: string;
```

Comment:
```suggestion
  accept?: string;
```
Can let this be optional in case we dont want to restrict type

Reply:
ok